src/Deposit.js

```diff
@@ -388,6 +388,14 @@ export default class Deposit {
     return toBN(await this.contract.methods.lotSizeSatoshis().call())
   }
 
+  /**
+   * Get the signer fee, to be paid at redemption.
+   * @return {Promise<BN>} A promise to the signer fee for this deposit, in TBTC.
+   */
+  async getSignerFee() {
```

Comment:
Although please include the unit in this method name, now that I think about it (`getSatoshiSignerFee` or similar).

Reply:
Almost Hungarian. 😉 

Observation - the naming styles are a bit diverse across the board, we seem to prefer lowercase Tbtc for getters, and uppercase for minting.

 - getSatoshiLotSize
 - getSignerFee
 - lotSizeTbtc
 - auctionTBTCAmount

Reply:
It's all a mess.